quattor/types/license.pan

```diff
@@ -0,0 +1,9 @@
+declaration template quattor/types/license;
+
+type structure_license = {
```

Comment:
s/license/licence/ 

license is the verb not the noun so I think this needs to change throughout the file. :(

Reply:
Ah, british english :smile: will run the search-and-replace ;-)